### PR TITLE
Update `namespace` property syntax

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Update the `namespace` property syntax according to the https://github.com/kubeshop/botkube/pull/686

## Related issue(s)

Fix https://github.com/kubeshop/botkube/issues/679

### CODE REVIEW

File: content/installation/Teams/_index.md [hunk 1]
```diff
@@ -192,11 +192,16 @@ We will use this TLS secret while deploying the BotKube backend.
                              # group/version/resource (G/V/R) format
                              # resource name should be plural
                              # (e.g apps/v1/deployments, v1/pods)
-        namespaces:          # List of namespaces, "all" will watch all the namespaces
+        namespaces:
```
Comment (pkosiec) [inline content/installation/Teams/_index.md:195]: 🤔 I thought we didn't want to update installation docs with 0.13 features? That was the main argumentation about merging https://github.com/kubeshop/botkube-docs/pull/112.

As I said previously, I'm fine with both approaches, as long as they are all-in:
- Have botkube.io deployed from different branch with full 0.12 documentation
- Have botkube.io deployed from main with all latest changes, even in installation docs (even with latest, as I assume the prerelease is around the corner).

But in the context of this PR I see third option:
WDYT about... removing these snippets? They are duplicates anyway, resource configurations should be in `source.md` doc and that's it.

Comment (mszostok) [reply content/installation/Teams/_index.md:195]: Yep, that's my mistake that I updated the installation doc, the should be. But I like the idea with DRY 👍 so let's remove them 🗑️

**Comment by mszostok** [comment]
Preview URL: https://update-ns-syntax.botkube-docs.pages.dev/installation/

File: content/installation/ElasticSearch/_index.md [hunk 2]
```diff
@@ -47,54 +47,15 @@ toc = true
...
-        fields:
-        - spec.template.spec.containers[*].image
-        - status.conditions[*].type
-  ```
-  - Pass the YAML file as a flag to `helm install` command, e.g.:
 
-    ```bash
-    $ helm install --version v0.12.4 --name botkube --namespace botkube --create-namespace -f /path/to/config.yaml --set=...other args..
-    ```
+  1. Create new file config.yaml and add resource configuration as described on the [source](/configuration/source) page.
```
Comment (pkosiec) [inline content/installation/ElasticSearch/_index.md:53]: Please add backticks:

```suggestion
  1. Create new file `config.yaml` and add resource configuration as described on the [source](/configuration/source) page.
```

I know Hugo theme renders them as items to copy, but with Docusaurus we won't have this problem 🙂

**Review by pkosiec — APPROVED** [review]

**Pull request merged**